.github/workflows/pypi-publish.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install setuptools wheel babel
+          pip install setuptools wheel babel importlib_metadata
```

Comment:
Is this needed?
This was [not added to the publish CI of `idutils` ](https://github.com/inveniosoftware/idutils/blob/master/.github/workflows/pypi-publish.yml#L23).

Reply:
why not updating it to 3.9 above instead>